### ISSUE TITLE
RAM usage climbs when adding movies and never goes back down

### ISSUE THREAD

**Issue by Gabriel-p** [jellyfin/jellyfin#3044]
**Describe the bug**
`jellyfin` is using almost 1 Gb of RAM when it is not being used. Appears to be related to `ffmpeg`.

Perhaps related to #2760?

**System (please complete the following information):**
 - OS: elementary OS 5.1.4 Hera (based on Ubuntu 18.04.4 LTS)
 - Virtualization: [e.g. Docker, KVM, LXC]
 - Clients: Browser
 - Browser: Chrome 81.0.4044.129 (Build oficial) (64 bits)
 - Jellyfin Version: 10.4.3
 - Playback: ?
 - Installed Plugins: [e.g. none, Fanart, Anime, etc.]
 - Reverse Proxy: [e.g. none, nginx, apache, etc.]
 - Base URL: none
 - Networking: ?
 - Storage: local

**To Reproduce**
Not sure. Leave the server open for a few days perhaps?

**Expected behavior**
Litttle to no RAM usage when idle.

**Logs**
The log shows many lines like these:

```
[2020-05-03 12:04:37.891 -03:00] [WRN] HTTP Response 200 to "127.0.0.1". Time (slow): 0:00:01.4405243. "http://127.0.0.1:8096/Items/583ff41e0075920e26f7172e8eb6859a/Images/Primary?maxHeight=960&maxWidth=640&tag=80f5b94f6970f13e6162c598016dddb3&quality=90"
[2020-05-03 12:04:38.038 -03:00] [WRN] HTTP Response 200 to "127.0.0.1". Time (slow): 0:00:01.1116095. "http://127.0.0.1:8096/Items/43002243e68a593daa90cf31b0419811/Images/Primary?maxHeight=960&maxWidth=640&tag=02c174ec067e5dd5a9c2b977d247f0fe&quality=90"
[2020-05-03 12:04:38.209 -03:00] [WRN] HTTP Response 200 to "127.0.0.1". Time (slow): 0:00:01.4937195. "http://127.0.0.1:8096/Items/412a368eeb6e5f9e8c916dfb2e8f89e5/Images/Primary?maxHeight=960&maxWidth=640&tag=b70ee3249930714978e37be8bfe3d921&quality=90"
[2020-05-03 12:04:38.335 -03:00] [WRN] HTTP Response 200 to "127.0.0.1". Time (slow): 0:00:01.518944. "http://127.0.0.1:8096/Items/2dd64842cfbf79769a9669d2d6d9e1e3/Images/Primary?maxHeight=960&maxWidth=640&tag=6c3053041a52a5da601268cc7a551c26&quality=90"
[2020-05-03 12:04:38.569 -03:00] [WRN] HTTP Response 200 to "127.0.0.1". Time (slow): 0:00:01.7128127. "http://127.0.0.1:8096/Items/d1fc84a19ca3545ea5200a5120bad257/Images/Primary?maxHeight=960&maxWidth=640&tag=b1fd2a30e4df6fa44572f090cae6b20d&quality=90"
[2020-05-03 12:04:38.618 -03:00] [WRN] HTTP Response 200 to "127.0.0.1". Time (slow): 0:00:01.0205263. "http://127.0.0.1:8096/Items/3ee0f677c57937318591cb4be60d536c/Images/Primary?maxHeight=960&maxWidth=640&tag=139de123add5a5d5d75bccbd1fe6cdc0&quality=90"
```

**Screenshots**
![Captura de pantalla de 2020-05-03 12-09-23](https://user-images.githubusercontent.com/6054951/80917753-74c4e580-8d37-11ea-8b96-58f485aec68f.png)

**Additional context**

**Comment by Artiume** [comment]
You're on 10.4.3. Can you update to latest?

**Comment by Gabriel-p** [comment]
@Artiume I wasn't aware I was using and older version. Shouldn't jellyfin ask me to update when a new version is released?

**Comment by Artiume** [comment]
Perhaps it will one day but not yet :)

**Comment by Gabriel-p** [comment]
Down to ~500 Mb when idle using the latest version (10.5.5). Is this the expected usage?

**Comment by Gabriel-p** [comment]
I've figured out where the issue is. When idle jellyfin sits around ~120 Mb, but as I start adding movies to the catalog it climbs a bit with each one added and never goes back down.

**Comment by Artiume** [comment]
Catalog?

**Comment by Gabriel-p** [comment]
@Artiume what do you mean?

**Comment by Artiume** [comment]
> but as I start adding movies to the catalog it climbs a bit with each one added

What do you mean by catelog?

**Comment by Gabriel-p** [comment]
Oh, I mean to the movies "media library", i.e., the folder I've set up to contain my movies.

**Comment by Artiume** [comment]
That could be due to using Real Time Monitoring. You could turn it off for a minute to see if that's causing the usage.

**Comment by Gabriel-p** [comment]
I tried turning it off and restarting the server. As I add movies to the dedicated folder and re-scan the library manually to add the item, the RAM usage increases and does not go back down once the movie is added.

**Comment by Artiume** [comment]
So #2760 is about web socket leakage, this RAM usage looks to more about focused on the sqlite database ram usage. How much content do you have with the 1 GB of ram usage?

**Comment by Gabriel-p** [comment]
> How much content do you have with the 1 GB of ram usage?

I'm sorry, I don't understand your question. Could you rephrase it please?

**Comment by Artiume** [comment]
How big are your libraries

**Comment by Gabriel-p** [comment]
212 movies (~200Gb) and 17 series (~150 Gb)

**Comment by Gabriel-p** [comment]
Hasn't happened again for a while
